src/Pages/Main/Component/Location/Location.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import { API } from '../../../../config';
+import styled from 'styled-components';
+
+const Location = ({ location, setLocation }) => {
+  const [isToggleBox, setToggleBox] = useState(false);
+  const [place, setPlace] = useState([]);
+
+  useEffect(() => {
+    fetch(`${API}/hotels/main`)
+      .then(res => res.json())
+      .then(res => {
+        setPlace(res.results.location);
+      });
+  }, []);
+
+  const handleToggleBox = () => {
+    setToggleBox(!isToggleBox);
+  };
+  const closeToggleBox = location => {
+    setToggleBox(false);
+    setLocation(location);
+  };
+
+  return (
+    <>
+      <LocationWrap onClick={handleToggleBox}>
+        <SearchBox value={location} />
+        <i class="fas fa-map-marker-alt" />
+        <i className="fas fa-chevron-down rightDown" />
+      </LocationWrap>
+      <SelectLocation display={isToggleBox}>
+        <Title>주변 인기 여행지</Title>
+        {place.map((location, index) => (
+          <SelectLocationTitle
+            onClick={() => {
+              closeToggleBox(`${location}`);
+            }}
+            value={location}
+          >
+            <i class="fas fa-map-marker-alt"></i>
+            {location}
+          </SelectLocationTitle>
+        ))}
+      </SelectLocation>
+    </>
+  );
+};
+
+export default Location;
+
+const LocationWrap = styled.div`
+  position: relative;
+  .fa-map-marker-alt {
+    position: absolute;
+    top: 17px;
+    left: 15px;
+  }
+  .rightDown {
+    position: absolute;
+    top: 15px;
+    right: 40px;
+  }
+  i {
```

Comment:
```suggestion
  }
  
  .rightDown {
    position: absolute;
    top: 15px;
    right: 40px;
  }
  
  i {
```
새로운 셀렉터 앞에서는 한줄 씩 띄워주세요...!

Reply:
새로운 셀렉터앞에는 한줄씩...수정하겠습니다!